File: Beecrowd/2963/2963.py
# Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/2963
N = int(input())
C = int(input())
elected = True
for i in range(N-1):
    v = int(input())
    if v > C:
        elected = False
if elected == True:
    print("S")
else:
    print("N")

File: Beecrowd/2963/2963.cpp
#include <iostream>
using namespace std;

int main(){
    int N, C, v;
    bool elected = true;
    cin >> N >> C;
    for(int i = 1; i < N; i++){
        cin >> v;
        if(v > C){ elected = false; }
    }
    if(elected == true){ cout << "S" << endl; }
    else{ cout << "N" << endl; }
 
    return 0;
}

